src/cocoa/toga_cocoa/widgets/divider.py

```diff
@@ -8,8 +8,6 @@
 class Divider(Widget):
     def create(self):
         self.native = NSBox.alloc().init()
-        self.native.interface = self.interface
```

Comment:
Is there any reason to take this one out? Is it because NSBox is a class being used 'as is' (rather than there being a custom Toga extension), so we can't define the weak property? We recently did a "consistency" pass to make sure that every `native` object had a reference to `interface` and `impl`, and it would be a pity to lose that consistency - although, I'll grant in this case there isn't a whole lot of utility.

Reply:
> Is it because NSBox is a class being used 'as is' (rather than there being a custom Toga extension), so we can't define the weak property?

Yes, exactly. If we cannot declare it as a weak reference, we are introducing reference cycles again.

On a side note, if you look at why access to the `impl` and `interface` layers is required functionally, they really are only required by the native widget's *delegate*. It just so happens that `toga_cocoa` subclasses the native classes, defines delegate methods on them, and sets the native objects to be their own delegates...